packages/calypso-color-schemes/src/shared/color-schemes/_aquatic.scss

```diff
@@ -138,6 +138,7 @@
 	/* Collapse Menu Button */
 	--color-collapse-menu-text: var(--color-sidebar-submenu-hover-text);
 
-	/* Command Palette Items */
-	--wp-admin-theme-color: var(--studio-celadon-50);
+	/* Gutenberg components */
+	--wp-admin-theme-color: var(--color-accent);
+	--wp-admin-theme-color-darker-20: var(--color-accent-60);
```

Comment:
Because we're now overriding this variable, and we weren't before, I was worried how many other places use `--wp-admin-theme-color-darker-20`. Maybe we'd unexpectedly update a colour that wasn't expecting to be `color-accent-60`?

But after searching I'm not worried anymore. Afaict this variable is hardly used anywhere else:
- [In the performance profiler link button style](https://github.com/Automattic/wp-calypso/blob/ab481c00e3a8a9d47b584bef005f2cf1fc777662/client/site-profiler/components/styles.scss#L75). However because the performance profiler is in the sites dashboard (not the site specific dashboard), the user's theme selection isn't even used.
- In the [suggested link popup](https://github.com/WordPress/gutenberg/blob/1339e85cbcc2769aa26d238fb491328ccfa2753a/packages/block-editor/src/components/url-input/style.scss#L72), in the block editor URL input component. But when testing the URL input component (e.g. adding a link to a paragraph) I didn't see any styles that looked broken. Even when I was using the Ecoplasm theme.

So I don't think we're going to have any issues setting the `--wp-admin-theme-color-darker-20` variable.